### DISCUSSION TITLE
changePageOnScroll params

### DISCUSSION THREAD

**Discussion by aamodio** [RussCoder/djvujs#58] [Q&A] [answered]
Hi,

trying to switch off the changePageOnScroll (with viewer 0.5.5 and lib 0.5.2) in this way but doesn't work:
viewer.configure({
    pageScale: 1,
    changePageOnScroll: false,
});

What's wrong?

Thanks,
A.

**Comment by RussCoder** [comment, marked as answer]
That is how it should be: 

```
viewer.configure({
  pageScale: 1,
  uiOptions: {
     changePageOnScroll: false,
  },
})
```

**Reply by aamodio** [reply]
Thanks!
